internal/kafka/kafka.go

```diff
@@ -171,11 +172,24 @@ func (b *kafkaBroker) Consume(ctx context.Context, topic string, handler func(ct
 			logger.Warn().Err(err).Msgf("Error when reading message: %s", err.Error())
 		} else {
 			logger.Trace().Bytes("payload", msg.Value).Msgf("Received message with key: %s", msg.Key)
+			ctx, err = ctxval.WithIdentityFrom64(ctx, returnHeaderValue(ctx, msg.Headers))
+			if err != nil {
+				logger.Trace().Msgf("Could not add identity to context: %s", err)
```

Comment:
This message does not tell you much, imagine you are reading logs because investigating an issue and read the message. Identity where? Is this HTTP context? Kafka context? I suggest:

```suggestion
				logger.Trace().Msgf("Could extract identity from context to Kafka message: %s", err)
```